Kiosk/Bid Fulfillment/ManualCreditCardInputViewController.swift

```diff
@@ -77,14 +77,19 @@ class ManualCreditCardInputViewController: UIViewController, RegistrationSubCont
         action
             .errors // Based on errors
             .take(1) // On the first error, then forever
+            .logNext()
             .mapReplace(with: false) // Replace the error with false
             .startWith(true) // But begin with true
             .bind(to: billingZipErrorLabel.rx_hidden) // show the error label
             .disposed(by: rx.disposeBag)
 
-        viewModel.moveToYear.take(1).subscribe(onNext: { [weak self] _ in
-            self?.expirationYearTextField.becomeFirstResponder()
-        }).disposed(by: rx.disposeBag)
+        viewModel.moveToYear
+            .take(1)
+            .observeOn(MainScheduler.asyncInstance)
```

Comment:
This was a weird one – RxSwift gave a really helpful error message here:

https://github.com/ReactiveX/RxSwift/blob/6b2a406b928cc7970874dcaed0ab18e7265e41ef/RxSwift/Rx.swift#L94-L104

Basically, in the `subscribe` closure, we are performing a side-effect that triggers another event to be triggered (importantly, it's triggered before the previous event finishes). `observeOn` is kind of a way to delay the invocation on to the next invocation of the runloop.